sdk/src/loom2d/Display/Stage.ls

```diff
@@ -292,6 +292,13 @@ package loom2d.display
         public native function get vectorQuality():int;
         public native function set vectorQuality(flags:int):void;
         
+        /**
+         * The recursion level of tessellation. Bigger values result in greater quality.
+         * Valid values are from 1 to 10.
```

Comment:
Can we note 6 is now default and 5 seems to result in errors? Also, note it's maximum level of recursion. (It is isn't it?)